chat.tandem.Client.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>chat.tandem.Client</id>
+  <provides><id>chat.tandem.Client.desktop</id></provides>
+  <launchable type="desktop-id">chat.tandem.Client.desktop</launchable>
+  <name>Tandem</name>
+  <developer_name>Tandem Communications Inc.</developer_name>
+  <summary>Virtual office for remote teams</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://tandem.chat</url>
+  <url type="help">https://intercom.help/tandem_help</url>
+  <description>
+    <p>Connect effortlessly with your distributed team. See who’s around, talk, and collaborate in one click.</p>
```

Comment:
Would be nice to have the description expanded. The marketing blurb is slightly longer on the website.

Reply:
I added all of the marketing "blurb", but maybe this is too much? `appstream-util` certainly thinks so. What would you include?

Reply:
At least some, current description is not much more over `<summary>`.

Reply:
How about that?

Reply:
LGTM